layers/+lang/haskell/config.el

```diff
@@ -17,7 +17,7 @@
 (spacemacs|defvar-company-backends haskell-cabal-mode)
 (spacemacs|defvar-company-backends intero-repl-mode)
 
-(defvar haskell-completion-backend 'ghci
+(defvar haskell-completion-backend `ghci
```

Comment:
Thanks 😸 Didn't know that you made a fix, no hurry 😸 Just wanted to make sure that it comes to develop without any issues 💃